app/Http/Controllers/ImageController.php

```diff
@@ -3,11 +3,41 @@
 namespace App\Http\Controllers;
 
 use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+
+use View;
 
 use App\Image;
+use App\Content;
+use Validator;
+use App\Http\Controllers\ContentController;
+
 
 class ImageController extends Controller
 {
 
+	public function store(Request $request)
+    {
+        $v = Validator::make($request->all(), [
```

Comment:
Just use

       $this->validate($request, [...rules...]);

See http://laravel.com/docs/5.1/validation#quick-writing-the-validation-logic